pkg/cmd/extension/command.go

```diff
@@ -293,19 +293,35 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 				Use:   "install <repository>",
 				Short: "Install a gh extension from a repository",
 				Long: heredoc.Docf(`
-					Install a GitHub repository locally as a GitHub CLI extension.
+					Install a GitHub CLI extension from a GitHub repository.
```

Comment:
I appreciate the cleanup of the language, here. I have a suggestion on how to make this even better

```suggestion
					Install a GitHub CLI extension from a GitHub or local repository.
```

The value prop here is that there are two ways to install an extension, which we call out here. In the comments further down, we specify which of the two ways we are providing instructions for and useful information about them.

What do you think?

Reply:
I like this improvement a lot, thanks @jtmcg ✨